src/styles/_breakpoints.scss

```diff
@@ -0,0 +1,26 @@
+$tablet-width: 768px;
+$desktop-width: 1024px;
+
+@mixin mobileOnly {
+  @media (max-width: #{$tablet-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin tabletOnly {
+  @media (min-width: #{$tablet-width}) and (max-width: #{$desktop-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin mobileAndTablet {
+  @media (max-width: #{$desktop-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin desktopOnly {
+  @media (min-width: #{$desktop-width}) {
+    @content;
+  }
+}
```

Comment:
I'm not a fan of this but I guess there's no choice since you cannot define your own `env()` variables :( So there's no way to customize it unless you compile the css in the project.

I think it's fine to include it like this and we can think about it...

Reply:
Ah, yeah, I totally forgot about those `env` variables... 😃  Let's implement that later.

Reply:
They won't work :(

> you cannot define your own env() variables

Only user agents cant for now. Afaik

Reply:
Getting there: https://caniuse.com/#feat=css-env-function